documentation/Set Up Mock Server/README.md

```diff
@@ -184,8 +184,9 @@ Select **Next**
             }
         ]
 ```
+**Note** : Kinldy remove the "c4c" from  locatingPolicy/rules/name:"sap.graph.*"/local
```

Comment:
Not a note, add this as a step as it needs to be executed every time.

8. In the `locating policy`, remove the `local` reference of `c4c` from the rule `sap.graph.*`. This will result in the rule as
```json
{
    "name": "sap.graph.*",
    "leading": "s4",
    "local": [
       
    ]
}
```